### PR TITLE
Reserve a top-level namespace 'contact-software'

### PR DESCRIPTION
**Pull request by wenns**
closes https://github.com/CycloneDX/cyclonedx-property-taxonomy/issues/100

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -35,7 +35,7 @@ interpreted as described in [RFC2119](https://datatracker.ietf.org/doc/html/rfc2
 
 ## Namespace Syntax
 
-Namespaces are hierarchical and delimited with a ":".
```
Comment (jkowalleck) [inline README.md:38]: this change was unintentional and must be reverted.

File: README.md [hunk 2]
```diff
@@ -65,7 +65,7 @@ ABNF syntax as per [RFC5234: Augmented BNF for Syntax Specifications: ABNF](http
 
 ## Registered Top Level Namespaces
 
-Regardless of other licensing attributes in this repository or document,
```
Comment (jkowalleck) [inline README.md:68]: this change was unintentional and must be reverted.

**Review by jkowalleck — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (wenns) [reply README.md:38]: Well, it was: I remoted trailing whitespace ;)
Same below. 

I can revert it, but I thought that you follow the "no trailing whitespace" convention?

Comment (wenns) [reply README.md:38]: Ah, I guess its the newline syntax in markdown... ?

Comment (jkowalleck) [reply README.md]: > I can revert it, but I thought that you follow the "no trailing whitespace" convention?


in markdown, 2 trailing white spaces are a line break.

**Review by jkowalleck — APPROVED** [review]

**Comment by jkowalleck** [comment]
merged, namespace is reserved.

Please drop a pull request as soon as you have your taxonomy published.

**Comment by jkowalleck** [comment]
hello @wenns , 

The reserved top level namespace `contact-software` exists for a while now, but the actual taxonomy was never linked here.
Could you check whether your taxonomy was published somewhere, and link it in the registry?

**Pull request merged**